File: magicdb-server/magicdb-script/magicdb-script-runtime/src/main/resources/examples/javascript/hello-world.js
// 简单的Hello World脚本
// 参数: name - 名称

// 返回问候消息
function greet(name) {
    return "Hello, " + name + "!";
}

// 执行并返回结果
var message = greet(name || "World");
message;

File: magicdb-server/magicdb-script/magicdb-script-runtime/src/main/resources/examples/python/hello-world.py
# 简单的Python Hello World脚本
# 参数: name - 名称

# 定义问候函数
def greet(name):
    return f"Hello, {name}!"

# 获取参数，如果没有提供则使用默认值
input_name = name if 'name' in globals() else "World"

# 执行并返回结果
greet(input_name)
